src/core/qunit-wrapper-nodejs-module.js

```diff
@@ -0,0 +1,42 @@
+// In a single Node.js process, if different parts or dependencies
+// of a project mix ESM and CJS, avoid a split-brain state by making
+// sure both import and re-use the same instance via this wrapper.
+//
+// Node.js 18+ can import a CommonJS file from ESM.
```

Comment:
Are you sure it's only 18+? From [12.x docs](https://nodejs.org/docs/latest-v12.x/api/esm.html#esm_import_statements):
> An `import` statement can reference an ES module or a CommonJS module.

Reply:
I only checked from 18 since that's our minimum, but fair enough, should have checked the docs for when it was introduced. Fixed.